app/src/utils/data/contributions.ts

```diff
@@ -7,6 +7,8 @@ import {
   GrantMetadata,
   GrantRound,
   GrantRoundMetadata,
+  GrantRoundDonation,
```

Comment:
(random comment location for threading)

Can we have the contributions sorted by timestamp, so the most recent ones are on top? Sort order seems to be based on transaction hash which isn't too intuitive

![image](https://user-images.githubusercontent.com/17163988/146278068-db2bf110-3d57-4ed5-a77c-a9ef3b8328a2.png)

Reply:
Just remembered that we want it sorted by amount (based on the issue). Should we sort by amount or timestamp?

Reply:
oh sure amount is fine if that's what was spec'd out!